docs/api.md

```diff
@@ -722,3 +722,66 @@ let client = Client.make(
 #### `composeExchanges`
 
 `composeExchanges` is a helper function that will compose a single exchange from an array of exchanges. Operations will be run through the provided exchanges in the order that they were provided to `composeExchanges`, from left to right.
+
+## `CombinedError`
+
+Errors in `reason-urql` are handed to your components and hooks via a record of type `CombinedError.t`. The record has the following type:
+
+```reason
+type combinedError = {
+  networkError: option(Js.Exn.t),
+  graphQLErrors: option(array(graphQLError)),
+  response: option(Fetch.response),
+  message: string,
+};
+```
+
+In this case, `networkError` returns the original JavaScript error thrown if a network error was encountered. `graphQLErrors` represent an `array` of errors of type `graphQLError`. These represent the errors encountered in the validation or execution stages of interacting with your GraphQL API. `response` is the raw `response` object returned by `fetch`. `message` is a stringified version of either the `networkError` or the `graphQLErrors` – `networkError` will take precedence.
+
+#### Example
+
+Dependening on the types of errors you get from your GraphQL API, you may want to do different things. Here's an example showing how to handle `networkError`s and `graphQLErrors` separately.
+
+```reason
+let ({ response }, _) = useQuery(~request, ());
+
+switch (response) {
+  | Error(e) =>
+    switch (e) {
+    | {networkError: Some(ne)} =>
+      <div>
+        {ne
+        ->Js.Exn.message
+        ->Belt.Option.getWithDefault("Network error")
+        ->React.string}
+      </div>
+    | {graphQLErrors: Some(gqle)} =>
+      <div>
+        {gqle
+        |> Array.to_list
+        |> List.map(e => {
+              let msg =
+                Belt.Option.getWithDefault(
+                  Js.Nullable.toOption(CombinedError.messageGet(e)),
+                  "GraphQL error",
+                );
+              "[GraphQLError: " ++ msg ++ "]";
+            })
+        |> String.concat(", ")
+        |> React.string}
+      </div>
+    | _ => <div> "Unknown error."->React.string </div>
+    }
+  }
+  | _ => <div />
+}
+```
+
+Due to the verbosity of the above, sometimes it isn't useful to handle every potential case around errors in your application. When you just want a simple error message of what went wrong to display to the user, just access the `message` field on the `CombinedError.t`:
+
+```reason
```

Comment:
For me this ends up being what I use 90% of the time, any thoughts on putting it before the more complex solution?

Reply:
It's also how I mainly handle errors in the front-end, I would also prefer if it was above the more complex way to handle errors just for the sake of it and to not confuse beginners

Reply:
Sounds good, thanks for the feedback!